### PR TITLE
Revise manuscript

### PR DESCRIPTION
**Pull request by mmore500**

### CODE REVIEW

File: joss/paper.md [hunk 1]
```diff
@@ -33,22 +33,25 @@ date: 10 June 2023
...
 This information reveals when traits were gained or lost, and also facilitates inference of underlying evolutionary dynamics [@mooers1997inferring;@dolsonInterpretingTapeLife2020;@moreno2024ecology].
 
-The Phylotrack project provides libraries for tracking and analyzing phylogenies in *in silico* evolution.
+Such capability is of great value to phylogenetics research. While backward-time models such as coalescent-based approaches are powerful, there are many scenarios that they cannot support, necessitating the use of forward-time evolutionary models [@hallerSLiMForwardGenetic2019]. Thus, research agendas that require predicting the phylogenetic patterns that would result from a given evolutionary process would benefit greatly from a way to track phylogenies in forward-time *in silico* evolution models or experiments. Currently, such tracking is implemented on a case-by-case basis for each individual digital model system. As efficient phylogenetic tracking is non-trivial to implement, this approach represents a research bottleneck.
```
Comment (mmore500) [inline joss/paper.md]:
```suggestion
Such capability is of great value to phylogenetics research.
While backward-time models such as coalescent-based approaches are powerful, there are many scenarios that they cannot support, necessitating the use of forward-time evolutionary models [@hallerSLiMForwardGenetic2019].
Thus, research agendas that require predicting the phylogenetic patterns that would result from a given evolutionary process would benefit greatly from a way to track phylogenies in forward-time *in silico* evolution. 
Currently, such tracking is implemented case-by-case in individual digital model systems.
As efficient phylogenetic tracking is non-trivial to implement, this *status quo* burdens research workloads.
```

**Review by mmore500 — COMMENTED** [review]
Looks good! Only a few minor counter-suggestions. Ready to merge, and dm me so I can pull to update the replit 👍 :)

**Comment by mmore500** [comment]
Wait, do we want to actually merge this now or leave this branch open while we submit? I assume we will eventually have to migrate from markdown to full latex.

File: joss/paper.md (hunk 1 above)
Comment (emilydolson) [reply joss/paper.md]: What if we just drop "simulations" from line 47 to side-step the whole instantiation/model/simulation/experiment philosophy of science discussion.

Comment (emilydolson) [reply joss/paper.md]: Otherwise looks good!

**Comment by emilydolson** [comment]
I think we will have to migrate to latex, but since we don't have to yet and this represents the most recent version of the paper (i.e. supersedes what is currently on main) we might as well merge it.

**Pull request merged**